commands/captureElementScreenshot.js

```diff
@@ -32,9 +32,24 @@ CaptureElementScreenshot.prototype.command = function command(
         promisifyCommand(api, 'getElementSize', [selector]),
         promisifyCommand(api, 'screenshot', [false])
     ]).then(([location, size, screenshotEncoded]) => {
-        const { x, y } = location
+        let { x, y } = location
         let { width, height } = size
 
+        /*
+         * Here we get the pixel density of the window and 
+         * ensure that we adjust the width and height accordingly
+         */
+        api.execute(function () {
+          return window.devicePixelRatio
+        }, [], function (devicePixelRatio) {
+          if (devicePixelRatio.value > 1) {
```

Comment:
I'd just remove this conditional. If the ratio === 1, then there will be no effect. Doing four product functions won't affect performance.

Reply:
There is no linting going on here, so I feel obligated to mention that indentation is 4 spaces. Thanks!